.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  # Automatically check for updates in framework CLIs for C3
+  - package-ecosystem: "npm"
+    directory: "/packages/create-cloudflare/src/frameworks"
+    schedule:
+      interval: "daily"
```

Comment:
can you please add a comment that this is temporary for us to test the system over the next week or two, and then we intend to switch over to weekly?

Reply:
comment added 🙂